Templates/ServiceBusTopicTrigger-NodeJS/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
+            "type": "serviceBusTrigger",
+            "direction": "in",
+            "topicName": "myinputtopic",
```

Comment:
Can you use the same topic name as the C# template? Why the arbitrary difference?